src/editor.py

```diff
@@ -149,14 +149,14 @@ def generate_code(self,dir,thread):
             if self.box_score: code = code + self.generate_boxscore(files)
             if self.line_score: code = code + self.generate_linescore(files)
             if self.scoring_plays: code = code + self.generate_scoring_plays(files)
-            if self.highlights: code = code + self.generate_highlights(files)
+            if self.highlights: code = code + self.generate_highlights(files,myteam,self.theater_link)

```

Comment:
I passed it in because the generate_highlights function does not otherwise differentiate between game and postgame threads. This seemed better than passing in the thread type and inserting another condition within the function, since there are different calls to the function from here.